test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -180,6 +194,15 @@ def host_setup_for_tensorflow_inference(serving_folder_path, framework_version,
         local_scripts_path = os.path.join("container_tests", "bin", "tensorflow_serving")
         ec2_connection.run(f"mkdir -p {serving_folder_path}")
         ec2_connection.run(f"cp -r {local_scripts_path} {serving_folder_path}")
+        if is_neuron:
+            neuron_local_model = os.path.join("$HOME", "container_tests/bin/neuron_tests/simple")
+            neuron_model_dir = os.path.join(serving_folder_path, f"models/")
```

Comment:
Please remove the `/` at the end of `f"models/"`. Also, `f"models/"` doesn't need to be an f-string, because it doesn't replace any variables in the string. It can be changed to `"models"`.